tools/test_configs/DefaultInterface.json

```diff
@@ -22,15 +22,13 @@
         "tcp-echo-prefix" : {
             "help" : "Some servers send a prefix before echoed message",
             "value" : null
-        },
-        "sim-blockdevice": {
-            "help": "Simulated block device, requires sufficient heap",
-            "macro_name": "MBED_TEST_SIM_BLOCKDEVICE",
-            "value": "HeapBlockDevice"
         }
     },
     "target_overrides": {
         "*": {
+            "nsapi.default-wifi-ssid": WIFI_SSID,
+            "nsapi.default-wifi-password": WIFI_PASSWORD,
+            "nsapi.default-wifi-security": WIFI_SECURITY,
```

Comment:
Is this missing " around the macro names?

Reply:
Yes, but my assumption was that these weren't used as-is, but were substituted before use by a `sed` in the test-scripts.

Reply:
Interesting. GitHub automatically indicated that they were missing quotes.

I think they should still have quotes. If a `sed` command were used, the user would still need to remember to add quotes.